transmogrifier/sources/xml/dspace_dim.py

```diff
@@ -259,6 +221,73 @@ def get_optional_fields(self, xml: Tag) -> dict | None:
 
         return fields
 
+    @classmethod
+    def get_contents(cls, xml: Tag) -> list[str]:
+        return [
+            str(contents.string)
+            for contents in xml.find_all(
```

Comment:
Per https://mitlibraries.atlassian.net/browse/TIMX-276, we should soon evaluate whether we want to use `lxml` or `BeautifulSoup` to parse XML